OFEP-sdk-e2e-test-strategy.md

```diff
@@ -0,0 +1,58 @@
+## OFEP: SDK end-to-end test strategy
+
+## State: Drafting
+
+This OFEP proposes to introduce a simplified process to write SDK end-to-end(e2e) tests
+
+## Background
+
+SDKs are at the core of OpenFeature as they contain language specific implementations of the OpenFeature specification [1]
```

Comment:
Please update the links to be footnotes. You can see the syntax here.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes

Reply:
This is cool - see the preview : https://github.com/open-feature/ofep/blob/2f7c045aebfe0c7ebed76354ac6f57628a91ad2c/OFEP-sdk-e2e-test-strategy.md#user-content-fn-2-b1b3f960f2d98fa55a3fcd378c954b6f